test/board_test.exs

```diff
@@ -48,7 +48,7 @@ defmodule BoardTest do
     token = :o
     opponent_token = :x
     assert capture_io(fn ->
-      Board.update(board, move, opponent_moves, token, opponent_token) end) ==
+      Board.render(board, move, opponent_moves, token, opponent_token) end) ==
```

Comment:
Why does `Board.render` require five arguments? Not saying you have to address this particular comment, but it does seem kind of weird to me...